app/views/courses/_users.json.jbuilder

```diff
@@ -4,7 +4,9 @@ json.users course.courses_users.eager_load(:user) do |cu|
   json.call(cu.user, :id, :wiki_id, :contribution_url, :sandbox_url)
   json.admin cu.user.permissions == User::Permissions::ADMIN
   json.recent_revisions RevisionStat.recent_revisions_for_user_and_course(cu.user, cu.course).count
-  json.course_training_progress ctp_manager.course_training_progress
+  unless ENV['disable_training'] == 'true'
+    json.course_training_progress ctp_manager.course_training_progress
```

Comment:
We should move all the training-related bits of this endpoint into the conditional.

```ruby
unless ENV['disable_training'] == 'true'
  ctp_manager = CourseTrainingProgressManager.new(cu.user, cu.course)
  json.course_training_progress ctp_manager.course_training_progress
  json.modules_overdue ctp_manager.first_overdue_module.present?
end